.github/workflows/ci.yml

```diff
@@ -7,8 +7,8 @@ jobs:
   pr:
     strategy:
       matrix:
-        node: [16.x] # tests both npm 7 & 8
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        node: [16.x]
+        os: [macos-latest, ubuntu-latest] # windows-latest is commented out because it breaks on adapter-node
```

Comment:
Do we have a JIRA to fix this? Or is it something we can't really control / work around?